MAINTENANCE.md

```diff
@@ -11,6 +11,21 @@ rules repos (this doc is not duplicated across the repos):
 The current maintainers can be seen in the [CODEOWNERS](CODEOWNERS)
 file.
 
+## How to become a maintainer
+
+These rules are community maintained. Any community member from any
+company can become a maintainer. Ideally we have a large set of
```

Comment:
I'd leave it at "community member" no need to be employed? Feels more inclusive.

Reply:
ah yep, sounds good thanks